smalltalksrc/VMMaker/SpurMemoryManager.class.st

```diff
@@ -6935,24 +6945,42 @@ SpurMemoryManager >> instantiateClass: classObj indexableSize: nElements [
 SpurMemoryManager >> instantiateClass: classObj indexableSize: nElements isPinned: isPinned [
 	<api>
 
+	^ self instantiateClass: classObj indexableSize: nElements isPinned: isPinned isOldSpace: false
+]
+
+{ #category : #instantiation }
+SpurMemoryManager >> instantiateClass: classObj indexableSize: nElements isPinned: isPinned isOldSpace: isOldSpace [
+	<api>
+
 	^ self subclassResponsibility
 ]
 
 { #category : #instantiation }
 SpurMemoryManager >> instantiateClass: classObj isPinned: isPinned [
+
+	^ self instantiateClass: classObj isPinned: isPinned isOldSpace: false
+]
+
+{ #category : #instantiation }
+SpurMemoryManager >> instantiateClass: classObj isPinned: isPinned isOldSpace: isOldSpace [
 	| instSpec classFormat numSlots classIndex newObj |
-	classFormat := self formatOfClass: classObj.
+	classFormat := self formatOfClassSafe: classObj.
```

Comment:
There is this change that I didn't do… I suppose that is a change that was done on `pharo-12`?

Reply:
Yes, we need to *not* do this change.